diagrams/library-gen.md

```diff
@@ -3,7 +3,7 @@ classDiagram
 
   class Book {
     +String title
-    +int pages
+    +Integer pages
```

Comment:
Not quite: I wanted to mirror the primitive types we support with their exact names, so with uppercase. Because these diagrams are the result of self-defining the M3, they should use the same names.